browser-test/src/admin_application_statuses.test.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  startSession,
+  logout,
+  loginAsGuest,
+  loginAsProgramAdmin,
+  loginAsAdmin,
+  selectApplicantLanguage,
+  ApplicantQuestions,
+  AdminPrograms,
+  userDisplayName,
+} from './support'
+import {Page} from 'playwright'
+
+describe('view program statuses', () => {
+  let pageObject: Page
+  const programWithoutStatusesName = 'test program without statuses'
+
+  beforeAll(async () => {
+    const {page} = await startSession()
+    pageObject = page
+  })
+
+  describe('without program statuses', () => {
+    beforeAll(async () => {
+      // Timeout for clicks and element fills. If your selector fails to locate
```

Comment:
Did you find that this call was necessary for the test? We only set it for one other test (presumably for a specific peculiarity of that test, though it doesn't appear to be well-documented).

If it's not needed right now, I'd prefer leaving it out until we encounter an issue. If we do leave it, I'd suggest documenting the specific selector / codepath that makes it necessary.